workshop/Lab1/README.md

```diff
@@ -1,46 +1,45 @@
 # Lab 1. Set up and deploy your first application
 
 Learn how to deploy an application to a Kubernetes cluster hosted within
-the IBM Container Service.
+the IBM Cloud Kubernetes Service.
 
-# 0. Install Prerequisite CLIs and Provision a Kubernetes Cluster
+## 1. Install the prerequisite CLIs and provision a Kubernetes cluster
 
 If you haven't already:
-1. Install the IBM Cloud CLIs and login, as described in [Lab 0](../Lab0/README.md).
+1. Install the IBM Cloud CLIs and log in, as described in [Lab 0](../Lab0/README.md).
 2. Provision a cluster:
 
    ```$ bx cs cluster-create --name <name-of-cluster>```
 
-Once the cluster is provisioned, the kubernetes client CLI `kubectl` needs to be
-configured to talk to the provisioned cluster.
-
-1. Run `$ bx cs cluster-config <name-of-cluster>`, and set the `KUBECONFIG`
-   environment variable based on the output of the command. This will
+   Once the cluster is provisioned, you need to configure the Kubernetes client CLI `kubectl`.
+   
+3. Run `$ bx cs cluster-config <name-of-cluster>` and set the `KUBECONFIG`
+   environment variable that is based on the output of the command. This will
    make your `kubectl` client point to your new Kubernetes cluster.
 
-Once your client is configured, you are ready to deploy your first application, `guestbook`.
+   Once your client is configured, you are ready to deploy your first application, `guestbook`.
 
-# 1. Deploy your application
+## 2. Deploy your application
 
-In this part of the lab we will deploy an application called `guestbook`
-that has already been built and uploaded to DockerHub under the name
+In this part of the lab, we deploy an application called `guestbook`
+that was built and uploaded to DockerHub under the name
 `ibmcom/guestbook:v1`.
 
-1. Start by running `guestbook`:
+1. Run `guestbook`by running the command:
 
    ```$ kubectl run guestbook --image=ibmcom/guestbook:v1```
 
    This action will take a bit of time. To check the status of the running application,
```

Comment:
Can we specify "a bit of time?" Such as, "This action can take x amount of minutes/hours, depending on xyz factors. It should not take 5 hours (or however amount of time)."